translations/binaries/slps_strings_items_consumable.json

```
@@ -0,0 +1,2153 @@
+            "original_bytes_length": 69,
+            "available_bytes_size": 71
+        },
+        {
+            "offset": 1759424,
+            "string": "[Sign_Box]レンズの結晶[END]",
+            "translation": "[Sign_Box]렌즈의 결정[END]",
+            "original_bytes_length": 18,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1759344,
+            "string": "加工されていないレンズの結晶。[LINE]\n非常に珍しいので高値で売れる[LINE]\n換金アイテム。[END]",
+            "translation": "가공되지 않은 렌즈의 결정.[LINE]\n매우 드물기 때문에 높은 가격에 팔린다.[LINE]\n교환 아이템.[END]",
+            "original_bytes_length": 75,
+            "available_bytes_size": 79
+        },
+        {
+            "offset": 1759328,
+            "string": "[Sign_Box]石ころ[END]",
+            "translation": "[Sign_Box]돌멩이[END]",
+            "original_bytes_length": 12,
+            "available_bytes_size": 15
+        },
+        {
+            "offset": 1759256,
+            "string": "１ガルドを笑う者は１ガルドに泣く。[LINE]\n一応１ガルドで売れる換金アイテム。[LINE]\n[END]",
+            "translation": "1가르드를 웃는 자는 1가르드에 울게 된다.[LINE]\n일단 1가르드에 팔린다.[LINE]\n[END]",
+            "original_bytes_length": 71,
+            "available_bytes_size": 71
+        },
+        {
+            "offset": 1759240,
+            "string": "[Sign_Box]天地鉱[END]",
+            "translation": "[Sign_Box]천지광[END]",
+            "original_bytes_length": 12,
+            "available_bytes_size": 15
+        },
+        {
+            "offset": 1759168,
+            "string": "天地戦争時代の建物に[LINE]\n使用されていたとされる鉱石。[LINE]\n換金アイテム。[END]",
+            "translation": "천지 전쟁 시대의 건물에[LINE]\n사용되었다고 알려진 광석.[LINE]\n교환 아이템.[END]",
+            "original_bytes_length": 65,
+            "available_bytes_size": 71
+        },
+        {
+            "offset": 1759152,
+            "string": "[Sign_Box]金香草[END]",
+            "translation": "[Sign_Box]금향초[END]",
+            "original_bytes_length": 12,
+            "available_bytes_size": 15
+        },
+        {
+            "offset": 1759080,
+            "string": "金の香りがする雑草。[LINE]\n金に対する執着心がよく分かる一品。[LINE]\n換金アイテム。[END]",
+            "translation": "금의 향기가 나는 잡초.[LINE]\n금에 대한 집착심이 잘 드러나는 제품.[LINE]\n교환 아이템.[END]",
+            "original_bytes_length": 71,
+            "available_bytes_size": 71
+        },
+        {
+            "offset": 1759064,
+            "string": "[Sign_Box]隕石の欠片[END]",
+            "translation": "[Sign_Box]운석의 조각[END]",
+            "original_bytes_length": 16,
+            "available_bytes_size": 16
+        },
+        {
+            "offset": 1758992,
+            "string": "天地戦争の発端たる隕石の欠片……[LINE]\nということにして売れる[LINE]\n換金アイテム。[END]",
+            "translation": "천지 전쟁의 발단이었던 운석의 조각……[LINE]\n으로 팔 수 있는[LINE]\n교환 아이템.[END]",
+            "original_bytes_length": 71,
+            "available_bytes_size": 71
+        },
+        {
+            "offset": 1758968,
+            "string": "[Sign_8]ポイズンマテリアル[END]",
+            "translation": "[Sign_8]포이즌 마테리얼[END]",
```

```suggestion
            "translation": "[Sign_8]포이즌 머티리얼[END]",
```

일본어 원문이 Material을 표기한 것이므로 Material의 한국어 표기인 머티리얼을 차용함